usaspending_api/etl/transaction_loaders/data_load_helpers.py

```diff
@@ -75,6 +75,9 @@ def get_deleted_fpds_data_from_s3(date):
     ids_to_delete = []
     regex_str = ".*_delete_records_(IDV|award).*"
 
+    if not date:
+        return []
```

Comment:
With most of our other functions, if a filter is not supplied, we fetch everything, not nothing. This could become a silent failure down the road.

Reply:
There was previous no check for date. Further down in the function the code assumes it is a well-formatted and valid datetime. If anything there should be more checks here